### DISCUSSION TITLE
Lowering minimum VSCode version to 1.80

### DISCUSSION THREAD

**Discussion by nfichter** [biomejs/biome#230] [General]
Is there a reason the minimum VSCode version is 1.81? Wondering if it's possible to lower to 1.80 (or the actual minimum necessary to run the Biome extension, whatever that is), since that's what [Cursor](https://www.cursor.so/) is built on, and I haven't been successful in asking them to upgrade to 1.81 yet.

**Comment by nhedger** [comment]
I'm pretty sure it could be lowered safely. The extension does not use any feature that has had major changes.

**Reply by Conaclos** [reply]
Feel free to open a PR :)

**Comment by nhedger** [comment]
https://github.com/biomejs/biome/pull/235

**Reply by nfichter** [reply]
Thanks!

**Reply by nhedger** [reply]
You're welcome!
